### DISCUSSION TITLE
bticino fc80gcs

### DISCUSSION THREAD

**Discussion by tibur79** [Koenkk/zigbee2mqtt#19203] [Devices]
the bticino fc80gcs device is not supported and does not release any values. Is it possible to solve the problem? Thank you

**Comment by xraver** [comment]
Hello
i think the device you are talking about is the legrand 412172 (https://www.zigbee2mqtt.io/devices/412172.html)
can you confirm this? if this is the case i can support to include also this one.

**Comment by tibur79** [comment]
hi, yes the device is Legrand

**Reply by xraver** [reply]
Ok, did you tried to pair it?

**Comment by tibur79** [comment]
yes yes certainly but it does not return values ​​and appears as unknownInviato da Alessandro Cenci Il giorno 29 gen 2024, alle ore 20:08, Giorgio Ravera ***@***.***> ha scritto:﻿
Ok, did you tried to pair it?

**Reply by xraver** [reply]
can you share your log? maybe it's related to [{hardwareVersion: XXX}, {hardwareVersion: XXX}]

**Comment by tibur79** [comment]
good morning, no I can't because it was restored with your app but I can tell you that it worked very well with ZHAInviato da Alessandro Cenci Il giorno 29 gen 2024, alle ore 22:01, Giorgio Ravera ***@***.***> ha scritto:﻿
can you share your log? maybe it's related to [{hardwareVersion: XXX}, {hardwareVersion: XXX}]

**Reply by xraver** [reply]
ok.
I have no chance in this moment to test this device. When you have chance to connect it back we can try to extend Koenkk app to include this device.
let me know if and when you are available!
thanks

ciao
